test/InterchainTokenService.js

```diff
@@ -195,6 +191,9 @@ describe('Interchain Token Service', () => {
             return [token, tokenManager, tokenId];
         };
 
+    deployFunctions.lockUnlock = deployNewLockUnlock;
+    deployFunctions.gateway = deployNewGateway;
+    deployFunctions.lockUnlockFee = deployNewLockUnlockFee;
```

Comment:
:+1:

Reply:
the code looks simpler and easier to understand. :)